mappings/net/minecraft/entity/ai/pathing/PathNodeMap.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_80 net/minecraft/entity/ai/pathing/PathNodeMap
```

Comment:
I think that this is pathNodeCache as seen in PathNodeNavigator which mightve been replaced by the usage of Int2ObjectMap from fastutil, the comments on this class will then be to match it closer to the Int2ObjectMap as seen in modern, this **SHOULD** very much be discussed as im heading into the game theory terriority here, but the prior possible existence of PathNodeNavigator and PathNodeMaker as a single class and the very specific usage of this class leads me to believe that im atleast on the right path

I think that the name PathNodeCache would be fitting

Reply:
field_860 is probably size